app/utils/bundle-importer.js

```diff
@@ -83,6 +83,23 @@ YUI.add('bundle-importer', function(Y) {
       this._executeDryRun(this.recordSet);
     },
 
+    _ensureV4Format: function(bundleYAML) {
+      try {
+        var bundle = jsyaml.safeLoad(bundleYAML);
+        // Return JSON.  JSON is a valid subset of YAML, and the internal
+        // JSON methods will be faster than relying on jsyaml.
+        if (bundle.services) {
```

Comment:
Good catch. I was matching Deployer behavior, but this allows for a basket named 'services'.